syntax = "proto2";

message XiguaLive {
	message Data {
		// WebcastChatMessage 聊天
		// WebcastGiftMessage 礼物
		required string method = 1;
		required bytes raw = 2;
	}
	repeated Data data = 1;
	required string cursor = 2;
	optional int32 fetch_interval = 3;
	optional int32 now = 4;
	required string internal_ext = 5;
}